css/ampdoc.css

```diff
@@ -62,8 +62,8 @@ html.i-amphtml-inabox-preserve-height-auto body {
 
 /** These properties can be overriden by user stylesheets. */
 body {
-  /* Text adjust is set to 100% to avoid iOS Safari bugs where the font-size is
-     sometimes not restored during orientation. See #449. */
+  /* Text adjust is set to 100% to avoid iOS Safari bugs in Viewer where
```

Comment:
@dvoytenko looked the issue, it was a fix for Viewer env, so I think we will not need it for inabox or FIE?

Reply:
I expect we don't need it in Safari, b/c ads are not normally responsive and orientation changes should not affect them as much.